### PR TITLE
Promote

### PR DESCRIPTION
**Pull request by jospoortvliet**
new and improved 'promote' page.
- added diaspora link (gotta show a little support for the open source social media, however irrelevant...)
- added ownCloud Insiders promo bit

### CODE REVIEW

File: page-promote.php [hunk 1]
```diff
@@ -1,24 +1,60 @@
...
+<div class="page-header">
+	<h1>Become an Insider</h1>
+</div>
+
+<!-- <div style="display: none;" id="insiders"> -->
+<!--	<p><strong>Thank you</strong> for using and supporting ownCloud.</p>
+
+	<p>We would like to invite you become an ownCloud Insider!</p>-->
+<!--	<h2>Insiders move forward</h2>
+	<p>Think of ownCloud Insiders as that place where connections are made, big ideas take off and magic happens. We reserved a seat at the table just for you!</p>-->
```
Comment (tomneedham) [inline page-promote.php]: @jospoortvliet What are these commented out sections for?

File: page-promote.php [hunk 2]
```diff
@@ -1,24 +1,60 @@
...
 		<h2>Recommend ownCloud</h2>
 		<p>Speak to friends, family, colleagues, or your existing service providers and recommend ownCloud.</p>
 	</div>
-</div>
+</div>-->
+<!--<script>
+  $( ".promote" ).click(function() {
+      $( "#insiders" ).show();
+  });
+</script>-->
```
Comment (tomneedham) [inline page-promote.php]: Likewise?

Comment (jospoortvliet) [inline page-promote.php]: hold-overs from my fscking around... I can remove them if you want

**Comment by tomneedham** [comment]
Fixed some scaling issues, but :+1:

**Comment by jospoortvliet** [comment]
Thanks a lot, @tomneedham @jancborchardt big step forward!

**Pull request merged**